spec/input_validator_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'input_validator'
+
+describe InputValidator do
+  let(:game_double) { Class.new { extend InputValidator } }
+
+  context 'when selection is an invalid character' do
```

Comment:
It's always good to check for different invalid and edge cases like nil, 0 and 10 for example.